src/auth/oauth/oauth.ts

```diff
@@ -24,6 +24,8 @@ import {
 } from '../../runtime/http';
 
 import {
+  SESSION_COOKIE_NAME,
```

Comment:
I noticed a complaint about a circular dependency here, so I moved these values to the types file.